### PR TITLE
locate Chrome the same way as chromedriver 

### LINKED ISSUE #45: Locate Chrome the same way as Google
Here is Google's code for how they get the default browser.
https://cs.chromium.org/chromium/src/chrome/test/chromedriver/chrome/chrome_finder.cc
This won't be an issue in 99% of cases, but it is possible that the webdrivers method and the chromedriver method can each get different information resulting in the wrong driver being used.

### PR DESCRIPTION
**Pull request by titusfortner**
Fixes #45 

I think I got this working the same was as Google.
It appears to work on Linux, Windows, JRuby & Mac for the testing environments we have at least.

Would it make sense to fall back to what we were doing before? chromedriver does not do anything with ENV or Registry like we were doing, though.

Since it's a potentially breaking change, the idea is to release this in a 4.0.alpha and get feedback from people as to whether it's working for them.

### CODE REVIEW

File: lib/webdrivers/chrome_finder.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          end
+        end
+      end
+
+      def win_version(location)
+        System.call "powershell (Get-ItemProperty '#{location}').VersionInfo.ProductVersion"
+      end
+
+      def linux_version(location)
+        System.call("#{Shellwords.escape location} --product-version").strip
```
Comment (kapoorlakshya) [inline lib/webdrivers/chrome_finder.rb]: `&.strip` could be helpful in preventing `NoMethodError (undefined method 'strip' for nil:NilClass)` if Chrome is installed in a custom location.

File: lib/webdrivers/chrome_finder.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        directories.each do |dir|
+          files.each do |file|
+            option = "#{dir}/#{file}"
+            return option if File.exist?(option)
+          end
+        end
+      end
+
+      def win_version(location)
+        System.call "powershell (Get-ItemProperty '#{location}').VersionInfo.ProductVersion"
```
Comment (kapoorlakshya) [inline lib/webdrivers/chrome_finder.rb]: Missing `strip`.

**Review by kapoorlakshya — APPROVED** [review]
I'm glad this is finally in place.

**Comment by titusfortner** [comment]
I'm going to hold off merging this until after a 3.9.x bug fixes

**Pull request merged**